game/src/debug/debug_plugin.rs

```diff
@@ -0,0 +1,15 @@
+use bevy::prelude::*;
+
+pub struct DebugPlugin;
+
+//add the system(s) you wish to use to see debug info
+impl Plugin for DebugPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_systems(Update, print_test);
+    }
+}
+
+//Function to test if debugger is added correctly
+fn print_test() {
+    info!("The debugger is working!");
```

Comment:
You should rename this function to something like "print_debug_active_message" or something like that. Even if the goal was just to test the system, I think having a message to indicate the debugger is running at the beginning of each run could be beneficial.

Reply:
Updated